### DISCUSSION TITLE
Is Copilot included with GitHub enterprise license and support

### DISCUSSION THREAD

**Discussion by varshagupt** [community/events#20] [Cloud]
Is Copilot included with GitHub enterprise license and support

**Comment by ossanna16** [comment]
Hi there @varshagupt! Currently Copilot is only available for personal accounts and not include in GitHub Enterprise.

**Comment by AtitBimali** [comment]
They have not mentioned anything regarding the GitHub Co-pilot for the GitHub enterprise license. You may have a look at this [page](https://enterprise.github.com/license) to know more about GitHub enterprise license and support

**Comment by rjbell4** [comment]
But Copilot for Business (or Copilot for Companies?) sounds like something that will support GitHub Enterprise (probably for an extra fee).

**Comment by bayu01** [comment]
The machine learning models and time needed to build usable results probably outweighs the benefits but I’m curious to know too!

**Comment by Bestra** [comment]
Here's a link to the [copilot for business waitlist](https://github.com/features/copilot/org_signup) that's linked from the keynote announcement [summary post](https://github.blog/2022-11-09-everything-new-from-github-universe-2022/)
